### PR TITLE
fix(brokeneck-fastify): solves 'Invalid options: Cannot find type User'

### LINKED ISSUE #171: graphql-tools is deprecated
After following the get started steps, trying to run `yarn start` on brokeneck-fastify is erroring

Steps to reproduce
1. Install according to the documentation.

2.  `cd packages/brokeneck-fastify/`, then `yarn start`

3. Error displayed
Screen Shot 2021-12-01 at 4 00 02 PM

Configuration
```
{
  "name": "@nearform/brokeneck-fastify",
  "version": "1.0.0-spinal.8",
  "author": {
    "name": "Simone Busoli",
    "email": "simone.busoli@nearform.com"
  },
  "repository": "nearform/brokeneck",
  "main": "lib/plugin.js",
  "scripts": {
    "create:env": "node -r fs -e \"fs.copyFileSync('.env.sample', '.env', fs.constants.COPYFILE_EXCL)\"",
    "dev": "yarn start",
    "start": "fastify start index.js -p 5001 -l debug -w",
    "test": "tap",
    "lint": "eslint .",
    "lint:fix": "eslint . --fix",
    "precommit": "lint-staged"
  },
  "dependencies": {
    "@azure/graph": "^5.0.1",
    "@azure/ms-rest-nodeauth": "^3.0.6",
    "@nearform/brokeneck-html": "^1.0.0-spinal.8",
    "auth0": "^2.30.0",
    "aws-sdk": "^2.793.0",
    "ejs": "^3.1.5",
    "env-schema": "^2.0.1",
    "fastify": "^3.8.0",
    "fastify-cli": "^2.5.1",
    "fastify-cors": "^6.0.0",
    "fastify-plugin": "^3.0.0",
    "fastify-static": "^4.0.0",
    "fluent-json-schema": "^3.0.0",
    "graphql": "^16.0.1",
    "graphql-iso-date": "^3.6.1",
    "graphql-tag": "^2.11.0",
    "graphql-tools": "^8.0.0",
    "mercurius": "^8.0.0",
    "pkg-dir": "^5.0.0",
    "point-of-view": "^4.7.0"
  },
  "devDependencies": {
    "babel-eslint": "^10.1.0",
    "eslint": "^7.13.0",
    "eslint-config-prettier": "^8.0.0",
    "eslint-config-standard": "^16.0.1",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-prettier": "^4.0.0",
    "faker": "^5.1.0",
    "lint-staged": "^12.1.1",
    "prettier": "^2.1.2",
    "proxyquire": "^2.1.3",
    "sinon": "^12.0.1",
    "tap": "^15.0.0"
  },
  "lint-staged": {
    "**/*.js": [
      "eslint --fix"
    ]
  }
}
```

### PR DESCRIPTION
**Pull request by gdorsi**
fixes #171

### CODE REVIEW

**Review by Eomm — APPROVED** [review]

**Review by simoneb — APPROVED** [review]
LGTM. Any idea why the build wouldn't break?

**Comment by gdorsi** [comment]
> LGTM. Any idea why the build wouldn't break?

The mercurius module is stubbed in the tests, so we have no feedback from there.

Anyway I'm not sure that the deprecation and the thrown error are related.

@zacharywenner could you please check if this PR fixes your issue?

**Comment by gdorsi** [comment]
The issue was with the graphql version.

I've downgraded it to v15 and added a test to check that the schema is loaded correctly

File: packages/brokeneck-fastify/lib/plugins/graphql/index.test.js [hunk 1]
```diff
@@ -24,3 +24,41 @@ tap.test('graphql', async t => {
...
+    `
+
+    const response = await fastify.inject({
+      method: 'POST',
+      headers: { 'content-type': 'application/json' },
+      url: '/graphql',
+      body: JSON.stringify({ query })
+    })
+
+    t.same(JSON.parse(response.body), {
```
Comment (simoneb) [inline packages/brokeneck-fastify/lib/plugins/graphql/index.test.js:56]: I think there's a response.json() method, I just don't remember if it returns a promise (I confuse it with the fetch api)

**Review by simoneb — APPROVED** [review]
LGTM, just one comment

File: packages/brokeneck-fastify/lib/plugins/graphql/index.test.js (hunk 1 above)
Comment (gdorsi) [reply packages/brokeneck-fastify/lib/plugins/graphql/index.test.js:56]: Thanks for the tip!

**Comment by zacharywenner** [comment]
> > LGTM. Any idea why the build wouldn't break?
> 
> The mercurius module is stubbed in the tests, so we have no feedback from there.
> 
> Anyway I'm not sure that the deprecation and the thrown error are related.
> 
> @zacharywenner could you please check if this PR fixes your issue?

@gdorsi Thanks! Works great. I'm up and running.

**Pull request merged**